client/src/css/Eesha.css

```diff
@@ -0,0 +1,264 @@
+  color: aliceblue;
+  font-size: 50px;
+  text-align: center;
+  padding-top: 7%;
+}
+
+.eesha-text {
+  font-family: 'PT Sans', cursive;
+  color: white;
+  text-align: center;
+  padding-top: 10px;
+}
+
+.eesha-text-unpadded {
+  font-family: 'PT Sans', cursive;
+  color: white;
+  padding-top: 2px;
+}
+
+.eesha-sign-chart-h1 {
+  font-family: 'PT Sans Caption', cursive;
+  color: #a19ab5  ;
+  padding: 1px;
+}
+
+.eesha-sign-chart-h2 {
+  font-family: 'PT Sans', cursive;
+  color: #bca4bc  ;
+  padding: 1px;
+}
+
+.icon {
+  position: relative;
+  margin-top: -7%;
+  margin-left: 7%;
+  width: 10%;
+}
+
+/* Callout box - fixed position at the bottom of the page */
+.callout {
+  bottom: 8%;
+  padding: -15%;
+  margin-left: 7%;
+  max-width: 300px;
+}
+
+/* Callout header */
+.callout-header {
+  font-family: 'PT Sans Caption', cursive;
+  padding: 5px 5px;
+  background:#F8D7D7  ;
+  font-size: 30px;
+  color: white;
+}
+
+/* Callout container/body */
+.callout-container {
+  padding: 5px;
+  background-color: #F8D7D7  ;
+  color: white
+}
+
+.flex-grid-container {
+  display: grid;
+  justify-content: space-between;
+  margin-top: -5%;
+  margin-left: 30%;
+  row-gap: 5px;
+}
+
+.flex-grid-container-cards {
+  display: grid;
+  justify-content: space-between;
+  margin-top: -5%;
+  margin-left: 30%;
+  margin-right: 10%;
+  row-gap: 5px;
+}
+
+.grid-one {
+  grid-column: 1;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-two {
+  grid-column: 2;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-three {
+  grid-column: 3;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-four {
+  grid-column: 1;
+  grid-row: 2;
+  width: 70%;
+}
+.grid-five {
+  grid-column: 2;
+  grid-row: 2;
+  width: 70%;
+}
+.grid-six {
+  grid-column: 3;
+  grid-row: 2;
+  width: 70%;
+}
+
+.grid-seven {
+  grid-column: 1;
+  grid-row: 3;
+  width: 70%;
+}
+
+.grid-eight {
+  grid-column: 2;
+  grid-row: 3;
+  width: 70%;
+}
+
+.grid-nine {
+  grid-column: 3;
+  grid-row: 3;
+  width: 70%;
+}
+
+.grid-1 {
+  grid-column: 1;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-2 {
+  grid-column: 2;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-3 {
+  grid-column: 3;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-4 {
+  grid-column: 4;
+  grid-row: 1;
+  width: 70%;
+}
+.grid-5 {
+  grid-column: 1;
+  grid-row: 2;
+  width: 70%;
+}
+.grid-6 {
+  grid-column: 2;
+  grid-row: 2;
+  width: 70%;
+}
+
+.grid-7 {
+  grid-column: 3;
+  grid-row: 2;
+  width: 70%;
+}
+
+.grid-8 {
+  grid-column: 4;
+  grid-row: 2;
+  width: 70%;
+}
+
+.grid-9 {
+  grid-column: 1;
+  grid-row: 3;
+  width: 70%;
+}
+
+.grid-10 {
+  grid-column: 2;
+  grid-row: 3;
+  width: 50%;
+}
+
+.grid-11 {
+  grid-column: 3;
+  grid-row: 3;
+  width: 70%;
+}
+
+.grid-12 {
+  grid-column: 4;
+  grid-row: 3;
+  width: 70%;
+}
+
+.grid-13 {
+  grid-column: 1;
+  grid-row: 4;
+  width: 70%;
+}
+
+.grid-14 {
+  grid-column: 2;
+  grid-row: 4;
+  width: 70%;
+}
+
+.grid-15 {
+  grid-column: 3;
+  grid-row: 4;
+  width: 70%;
+}
+
+.grid-16 {
```

Comment:
this is important: you should never need to use this many different classes for a grid. each card/image usually has the same styling in a grid. the only difference between them are their relative placement in the grid. if you use flexbox, you really only need one className for each of these 16 cards, and they'll automatically be laid out in order